src/pkcs15init/muscle.profile

```diff
@@ -8,7 +8,7 @@ cardinfo {
 
     max-pin-length	= 8;
     min-pin-length	= 4;
-    pin-encoding	= ascii-numeric;
+	pin-pad-char   = 0x00;
```

Comment:
Is there something not working with the current setup?

Reply:
see above